packages/frontend-shared/src/components/StandardModalHeader.vue

```diff
@@ -35,10 +37,13 @@ import { useI18n } from '@cy/i18n'
 
 const { t } = useI18n()
 
-defineProps<{
+withDefaults(defineProps<{
   helpLink: string
   helpText: string
-}>()
+  noHelp?: boolean
+}>(), {
+  noHelp: false,
```

Comment:
Because this defaults to false, existing modals that don't pass a `helpLink` prop will have links in the modal title that are dead. Thanks @marktnoonan for pointing this out

Reply:
Glad you checked! Meanwhile I've edited my comment to remove this line:

> In actual usage those components are doing what is expected as far as I can tell, so not a blocker.

Because I was checking on a different branch.